apps/ledger-live-desktop/src/renderer/components/LiveAppDrawer.tsx

```diff
@@ -85,8 +85,11 @@ const INCOMPATIBLE_NANO_S_CURRENCY_KEYS: Keys = {
   },
 };
 const getIncompatibleCurrencyKeys = (exchange: ExchangeSwap) => {
-  const parentFrom = exchange?.fromParentAccount?.currency?.id || "";
-  const parentTo = exchange?.toParentAccount?.currency?.id || "";
+  const parentFrom =
+    (exchange?.fromAccount?.type === "TokenAccount" && exchange?.fromParentAccount?.currency?.id) ||
+    "";
+  const parentTo =
+    (exchange?.toAccount?.type === "TokenAccount" && exchange?.toParentAccount?.currency?.id) || "";
```

Comment:
If we want we could also use `exchange?.fromAccount?.currency.parentCurrency` instead of `exchange?.fromParentAccount?.currency?.id` but it's fine like that